sale_automatic_workflow/models/sale.py

```diff
@@ -41,3 +41,14 @@ def onchange_workflow_process_id(self):
             warning = {'title': _('Workflow Warning'),
                        'message': workflow.warning}
             return {'warning': warning}
+
+    @api.multi
+    def action_invoice_create(self, grouped=False, final=False):
+        for order in self:
+            if not order.workflow_process_id.invoice_service_delivery:
+                continue
+            for line in order.order_line:
+                if line.qty_delivered_updateable and not line.qty_delivered:
```

Comment:
I should have checked what was `qty_delivered_updateable` (name is not self-explanatory). Sorry for the trouble.